src/app/app.module.ts

```diff
@@ -17,10 +19,11 @@ import { SharedModule } from './shared/shared.module';
     FontAwesomeModule,
     LoginModule,
     RegistrationModule,
-    SharedModule
-
-
+    SharedModule,
+    appRouting,
+    AuthModule
```

Comment:
Don't you have an error with HttpClient provider ? I had to import HttpClientModule here